certora/harness/MarketplaceHarness.sol

```diff
@@ -7,12 +7,22 @@ import {IGroth16Verifier} from "../../contracts/Groth16.sol";
 import {MarketplaceConfig} from "../../contracts/Configuration.sol";
 import {Marketplace} from "../../contracts/Marketplace.sol";
 import {RequestId, SlotId} from "../../contracts/Requests.sol";
+import {Requests} from "../../contracts/Requests.sol";
 
 contract MarketplaceHarness is Marketplace {
-    constructor(MarketplaceConfig memory config, IERC20 token, IGroth16Verifier verifier) Marketplace(config, token, verifier) {}
+    constructor(MarketplaceConfig memory config, IERC20 token, IGroth16Verifier verifier)
+        Marketplace(config, token, verifier)
+    {}
 
     function publicPeriodEnd(Period period) public view returns (uint256) {
         return _periodEnd(period);
     }
-}
 
+    function slots(SlotId slotId) public view returns (Slot memory) {
+        return _slots[slotId];
+    }
```

Comment:
We had this harness API before and then it was removed in #165 
I had a discussion with the Certora team about this. Specifically discussing, when it makes sense to favour ghost variables over functions like these to keep the harness surface low.

I don't think there's a good reason not to have these functions as long as you don't change the logic of the original contract, so I think this is fine.

Otherwise, you could introduced ghost variables for the fields you're interested in.